src/plugins/inputs/part.js

```diff
@@ -9,26 +9,43 @@ exports.commands = ["close", "leave", "part"];
 exports.allowDisconnected = true;
 
 exports.input = function(network, chan, cmd, args) {
-	if (chan.type === Chan.Type.LOBBY) {
+	let target, partedChan;
+	if (args.length === 0) {
+		target = chan.name;
```

Comment:
Could we simplify this and only have 1 variable, because we are just storing the name twice essentially.

So something like
```
const target = args.length === 0 ? chan : _.find(network.channels, {name: args[0]});
```

And then when we want to refer to the name, we can just do `target.name`. Does that make sense?

Reply:
All right!